reference/tax_zone_check.v3.yml

```diff
@@ -0,0 +1,184 @@
+openapi: '3.0.0'
+info:
+  title: Tax Zone Check
+  description: Check what tax zone a given address and customer group derives to.
+  termsOfService: 'https://www.bigcommerce.com/terms'
+  contact:
+    name: BigCommerce
+    url: 'https://www.bigcommerce.com'
+    email: support@bigcommerce.com
+  version: ''
+security:
+  - X-Auth-Token: [ ]
+tags:
+  - name: Tax Zone Check
+servers:
+  - url: 'https://api.bigcommerce.com/stores/{store_hash}/v3'
+    variables:
+      store_hash:
+        default: store_hash
+        description: Permanent ID of the BigCommerce store.
+    description: BigCommerce API Gateway
+paths:
+  "/tax/zonecheck":
+    parameters:
+      - $ref: '#/components/parameters/Accept'
+    post:
+      tags:
+        - Tax Zone Check
+      summary: Check zone given an address.
+      description: Check what tax zone a given address and customer group derives to.
```

Comment:
```suggestion
  description: Check the applicable tax zone for a given address and customer group.
```

Reply:
I see the comment is resolved but the description is the same as previously?

Reply:
I think I got that mixed up with the changes on line 4

Reply:
updating, ty, good catch